staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go

```diff
@@ -158,11 +162,46 @@ func getExcludedChecks(r *http.Request) sets.String {
 	return sets.NewString()
 }
 
+type verbosityLevel int
+
+const (
+	quiet verbosityLevel = iota
+	verbose
+	duration
```

Comment:
I was surprised to see a verbosity level named by a field that is output at that level. Usually verbosity levels are named based on severity (info, warn, error, ...) or numbered (like kube-apiserver has), but not named after a field that is included in output for that level.